src/DssSpell.sol

```diff
@@ -35,196 +36,165 @@ contract DssSpellAction is DssAction {
+    // Dai Foundation Core Unit (Emergency Fund)
+    address constant DAIF_EF_WALLET = 0x5F5c328732c9E52DfCb81067b8bA56459b33921f;
+    // GovAlpha Core Unit
+    address constant GOV_WALLET     = 0x01D26f8c5cC009868A4BF66E268c17B057fF7A73;
+    // Growth Core Unit
+    address constant GRO_WALLET     = 0x7800C137A645c07132886539217ce192b9F0528e;
+    // Marketing Content Production Core Unit
+    address constant MKT_WALLET     = 0xDCAF2C84e1154c8DdD3203880e5db965bfF09B60;
+    // Oracles Core Unit
+    address constant ORA_WALLET     = 0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf;
+    // Protocol Engineering
+    address constant PE_WALLET      = 0xe2c16c308b843eD02B09156388Cb240cEd58C01c;
+    // Risk Core Unit
+    address constant RISK_WALLET    = 0xd98ef20520048a35EdA9A202137847A62120d2d9;
+    // Real-World Finance Core Unit
+    address constant RWF_WALLET     = 0x9e1585d9CA64243CE43D42f7dD7333190F66Ca09;
+    // Ses Core Unit
+    address constant SES_WALLET     = 0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6;
+
+    uint256 constant MAY_01_2021 = 1619827200;
+    uint256 constant JUN_21_2021 = 1624233600;
+    uint256 constant JUL_01_2021 = 1625097600;
+    uint256 constant SEP_01_2021 = 1630454400;
+    uint256 constant SEP_13_2021 = 1631491200;
+    uint256 constant SEP_20_2021 = 1632096000;
+    uint256 constant OCT_01_2021 = 1633046400;
+    uint256 constant NOV_01_2021 = 1635724800;
+    uint256 constant JAN_01_2022 = 1640995200;
+    uint256 constant MAY_01_2022 = 1651363200;
+    uint256 constant JUL_01_2022 = 1656633600;
+    uint256 constant SEP_01_2022 = 1661990400;
+
+    uint256 constant MILLION = 10 ** 6;
+    uint256 constant WAD     = 10 ** 18;
+
+    // Turn off office hours
+    function officeHours() public override returns (bool) {
+        return false;
+    }
 
     function actions() public override {
+        // Fix PAX keys
+        DssExecLib.setChangelogAddress("PAX", DssExecLib.getChangelogAddress("PAXUSD"));
+        DssExecLib.setChangelogAddress("PIP_PAX", DssExecLib.getChangelogAddress("PIP_PAXUSD"));
+        ChainlogLike(DssExecLib.LOG).removeAddress("PIP_PSM_PAX");
+
+        // Set unique payments
+        DssExecLib.sendPaymentFromSurplusBuffer(DAIF_WALLET,    2_000_000);
+        DssExecLib.sendPaymentFromSurplusBuffer(DAIF_EF_WALLET,   138_591);
+        DssExecLib.sendPaymentFromSurplusBuffer(SES_WALLET,       155_237);
+
+        // Setup both DssVest modules
+        DssExecLib.authorize(DssExecLib.vat(), MCD_VEST_DAI);
+        DssExecLib.authorize(DssExecLib.getChangelogAddress("GOV_GUARD"), MCD_VEST_MKR);
+        DssVestLike(MCD_VEST_DAI).file("cap", 1 * MILLION * WAD / 30 days);
+        DssVestLike(MCD_VEST_MKR).file("cap", 1_100 * WAD / 365 days);
+        DssExecLib.setChangelogAddress("MCD_VEST_DAI", MCD_VEST_DAI);
+        DssExecLib.setChangelogAddress("MCD_VEST_MKR", MCD_VEST_MKR);
+
+        // Set DAI stream payments
+        DssVestLike(MCD_VEST_DAI).restrict(
```

Comment:
Probably don't need to do it for all of these, but for at least the first one consider breaking this up and adding comments so that it's easier for auditors to grok.

```
DssVestLike(MCD_VEST_DAI).restrict(    // Only recipient can request funds
    DssVestLike(MCD_VEST_DAI).create(                                
        COM_WALLET,                    // Recipient of vest
        122_700.00 * 10**18,           // Total token amount of vest over period
        SEP_01_2021,                   // Start timestamp of vest
        JAN_01_2022 - SEP_01_2021,     // Duration of the vesting period (in seconds)
        0,                             // Length of cliff period (in seconds)
        address(0))                    // Manager address
    );
```

Reply:
I think I prefer to put it as a comment or link to the source instead of having a vesting which doesn't align with the rest.

Reply:
Comment looks good